src/mlpack/methods/neighbor_search/knn_main.cpp

```diff
@@ -63,11 +63,12 @@ PARAM_INT_IN("k", "Number of nearest neighbors to find.", "k", 0);
 
 // The user may specify the type of tree to use, and a few parameters for tree
 // building.
-PARAM_STRING_IN("tree_type", "Type of tree to use: 'kd', 'vp', 'cover', 'r', "
-    "'r-star', 'x', 'ball', 'hilbert-r', 'r-plus', 'r-plus-plus'.", "t", "kd");
-PARAM_INT_IN("leaf_size", "Leaf size for tree building (used for kd-trees, vp "
-    "trees, R trees, R* trees, X trees, Hilbert R trees, R+ trees and R++ "
-    "trees).", "l", 20);
+PARAM_STRING_IN("tree_type", "Type of tree to use: 'kd', 'vp', 'rp-tree', "
+    "'max-split-rp-tree', 'cover', 'r', 'r-star', 'x', 'ball', 'hilbert-r', "
+    "'r-plus', 'r-plus-plus'.", "t", "kd");
```

Comment:
We should call it `rp` and `max-rp` to be consistent with the rest of the tree names.